net/pagekitec/Makefile

```diff
@@ -8,16 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pagekitec
-PKG_REV:=0.91.201110
-PKG_VERSION:=$(PKG_REV)C
+PKG_VERSION:=0.91.201110
 PKG_RELEASE:=2
 PKG_LICENSE:=Apache-2.0
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
These can be removed.

Reply:
You mean PKG_SOURCE_SUBDIR and PKG_SOURCE, or what?

Reply:
both.